xla/service/gpu/runtime/topk_kernel_test.cc

```diff
@@ -72,6 +72,16 @@ std::vector<T> RandomVecNegative(int num_elements) {
 PrimitiveType Get(float) { return PrimitiveType::F32; }
 PrimitiveType Get(Eigen::bfloat16) { return PrimitiveType::BF16; }
 
+se::StreamExecutor *GetGpuExecutor() {
+  se::Platform* platform =
+#if GOOGLE_CUDA
```

Comment:
You could use GpuPlatformName() to avoid the #if #else here.

https://github.com/openxla/xla/blob/main/xla/stream_executor/gpu/gpu_init.h#L39

Reply:
good point! thank you, I have fixed it